sdk/servicebus/service-bus/test/stress/stress-test-resources.json

```diff
@@ -0,0 +1,76 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
```

Comment:
This is still checked in and the bicep is missing (assuming this was not the intention based on the latest gitignore update)

Reply:
good catch, removed.
Bicep is already present.